wire-ios-request-strategy/Sources/Request Strategies/Client Message/ClientMessageRequestStrategy.swift

```diff
@@ -144,22 +145,27 @@ extension ClientMessageRequestStrategy: InsertedObjectSyncTranscoder {
                         }
                     }
                 }
+                messageExpirationTimer.stop(for: object)
+                managedObjectContext.enqueueDelayedSave()
             }
 
             completion()
+            managedObjectContext.leaveAllGroups(groups)
```

Comment:
Shouldn't `completion()` be the very last call? I'm just asking, I don't have a clear picture yet.

Reply:
The `leaveAllGroups` is only used in the the tests in order to wait until operations are done so in this case it's fine to do after the `completion()` has been called.